### ISSUE TITLE
Anserini near-term roadmap - v0.3.0 and v0.4.0

### ISSUE THREAD

**Issue by lintool** [castorini/Anserini#484]
Mostly for @Peilin-Yang and @lukuang but others are most definitely welcome to join in discussion.

We're currently at v0.2.0 - here's my proposal for the near-term roadmap:
+ v0.3.0 - build regressions for TREC 2018 results.
+ v0.4.0 - upgrade to Lucene7

Rationale is that I would feel more comfortable about repeatability and replicability only after we get regression in place, and I want that before moving to Lucene7, since the regressions numbers will all change.

**Comment by lukuang** [comment]
I agree. Then we need to decide which tracks/tasks to be included and what should be the queries, e.g. queries for news track.

**Comment by lintool** [comment]
Ellen tells me that topics/qrels for TREC 2018 are sequestered until Feb 2019 - so we can't post publicly and check into repo.

This means that whatever we do, we need to include a note that says - "to replicate these results, download the official topics and qrels from NIST, rename, and place into `/some/path/`"

**Comment by Peilin-Yang** [comment]
@lintool 
> "to replicate these results, download the official topics and qrels from NIST, rename, and place into /some/path/" 

I think you meant topic files only? 
If the qrels files are available then we can download and put into the repo?

**Comment by lukuang** [comment]
> @lintool
> 
> > "to replicate these results, download the official topics and qrels from NIST, rename, and place into /some/path/"
> 
> I think you meant topic files only?
> If the qrels files are available then we can download and put into the repo?

I think both? Since these files might be available to us since we are participants, but not available to the public. Thus, we cannot include them in the repo until next Feb.

**Comment by lintool** [comment]
Both.

**Comment by Peilin-Yang** [comment]
Unfortunately, this (asking the users to download the topics/qrels files) might be our best bet for now

**Comment by lintool** [comment]
Closing, superseded by #579
